key-terms.md

```diff
@@ -1,211 +1,194 @@
+WCAG2ICT provides some key glossary terms to address differences between web and non-web contexts and to introduce terms that are nonexistent in WCAG but important to define for a non-web context.  “Content” and “user agent” are glossary terms from WCAG 2 that need to be interpreted significantly differently when applied to non-web ICT.  The glossary term “Web page” in WCAG 2 is replaced with newly defined terms “document” and “software”, and both “set of web pages” and “multiple web pages” are replaced with the newly defined terms “set of documents” and “set of software programs”.  The new terms introduced by WCAG2ICT are “accessibility services of platform software” because non-Web software doesn’t leverage the WCAG notion of a user agent, and "closed functionality" which is specific to non-web software.  The remaining glossary terms from WCAG 2 are addressed in [Chapter 7 Comments on Definitions in WCAG 2 Glossary](#comments-on-definitions-in-wcag-2-glossary).
+
+Terms defined and used in WCAG2ICT are applicable only to the interpretation of the guidance in this document.  The particular definitions should not be interpreted as having applicability to situations beyond the scope of WCAG2ICT.  Further information on usage of these terms follows.
 
 ### Accessibility Services of Platform Software
 
 The term **accessibility services of platform software**, as used in WCAG2ICT, has the meaning below:
 
-<DL><DT>accessibility services of platform software (as used in WCAG2ICT)</DT><DD>
+> services provided by an operating system, [user agent](#user-agent), or other platform software that enable non-web [documents](#document) or [software](#software) to expose information about the user interface and events to assistive technologies and accessibility features of software
```

Comment:
This isn't a quote of WCAG, so quote notation ">" should not be used. It was my understanding when Michael Cooper worked on converting this document that any definition should use the "<dl><dt> <dd> HTML so I don't think any of these changes are needed.

Reply:
The rendered pages had each DT as their own DL.  But it might shake out fine with the formal publications....